app/models/event.rb

```diff
@@ -1,12 +1,46 @@
 class Event < ApplicationRecord
+  include ApplicationHelper
+
   validates :title, presence: true, length: { maximum: 50 }
   validates :description, presence: true, length: { maximum: 200 }
   validates :start_at, presence: true
   validates :venue, presence: true, length: { maximum: 50 }
   validates :source_link, presence: true
 
+  after_create_commit :notify
+
+  class << self
+    def notify_upcoming_events
+      start_of_today = Time.current.beginning_of_day
+      end_of_two_days_later = Time.current.days_since(2).end_of_day
```

Comment:
「今日のはじまり」と「明後日のおわり」を強調する変数名として、こんなものはどうか？の提案です :bulb:

```suggestion
      beginning_of_today = Time.current.beginning_of_day
      end_of_day_after_tommorow = 2.days.since.end_of_day
```

Reply:
たしかに、`start`は名詞ですが動詞でもあるので、どちらかといえば"始める"というニュアンスを含むのに対して、`begginnng`は"始まり"感がありますもんね。
明後日も専用の英語を使った方がぱっと意味が伝わるのでコミット取り込みます〜!
ありがとうございます

Reply:
メソッド名が beginning_of_day なので、すなおに beginning_of な名前にしておくと対応も取れていいだろう〜と考えました :wave: